### PR TITLE
Release 0.16.0

### PR DESCRIPTION
**Pull request by klautcomputing**
Let's release a new version!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
 ### 0.16.0
-
 **Important:** This release changes the officially supported Kubernetes versions to v1.7 through v1.9. Other versions may continue to work, but we are no longer running our test suite against them.
 
+*Features*
```
Comment (dturn) [inline CHANGELOG.md]: Its a nit but this has been `Enhancements` in all previous versions, is the change intentional?

Comment (KnVerey) [reply CHANGELOG.md]: I originally intended for there to be three things: Features (new thing not previously possible), Enhancements (existing thing made better) and Bug Fixes. Looks like we only used "Features" in 0.13.0 so far though. Do you think it's a useful distinction?

Comment (klautcomputing) [reply CHANGELOG.md]: Intentional because of e.g:

```
### 0.13.0
*Features*
- Added support for StatefulSets for kubernetes 1.7+ using RollingUpdate
```

Comment (dturn) [reply CHANGELOG.md]: I like the distinction.  tI just wanted to make sure we were being consistent.

**Pull request merged**